developers/contributing_how_tos.md

```diff
@@ -0,0 +1,41 @@
+# How to contribute a napari How-To
+
+This should be a sentence (or brief paragraph) about what this how-to will achieve and how it helps address a common problem/task/need.
+
+## Prerequisites
+
+- this template
+- a list of what the user needs to prepare to be ready to start this how-to guide 
+- links to any existing napari.org tutorials or how-to guides that can help the user fulfill these prerequisites
+- the level of python/napari knowledge required to follow this how-to
+    - try to be specific about what skills are needed e.g. 
+    'connecting callbacks to layer events' or 'using matplotlib to produce plots'
+- plugins that should be installed
+- python packages that should be installed (don't list napari dependencies)
+
+***
+
+## How-to steps
+
+Headings in this section are yours to decide on. You should split your how-to guide into easy to follow, sequential steps to achieve the stated goal. Remember that your how-to may be used as a reference, so your headings should be descriptive and allow users to get a quick overview of the document, and which steps are relevant to them.
+
+## Include pictures
+
+You should include pictures/videos, particularly when describing interactions with the viewer. If an action can be performed both through the viewer and through code, you should include both options.
```

Comment:
<3

Reply:
(should we include a mention of accessibility - "try to use descriptive and complete alt-text when including images"?)